examples/v1alpha3/random-example.yaml

```diff
@@ -26,13 +26,15 @@ spec:
           spec:
             template:
               spec:
+                serviceAccountName: metrics-collector
```

Comment:
I am not sure why do we need to set service account for the trial

Reply:
for now, metrics-collector get logs from k8s api, it need some role. I will delete it after implement file reading

Reply:
SGTM